crates/fluvio/src/producer/partition_producer.rs

```diff
@@ -24,6 +28,10 @@ use super::{
 use super::accumulator::{BatchEvents, BatchesDeque};
 use super::event::EventHandler;
 
+pub const BACKOFF_MIN_DURATION: Duration = Duration::from_secs(1);
+pub const BACKOFF_MAX_DURATION: Duration = Duration::from_secs(30);
+pub const BACKOFF_FACTOR: f64 = 1.1;
+
```

Comment:
Should these use common values to the consumer side of things, and maybe be in `fluvio-types/src/defaults.rs`?

Reply:
Sure! I moved the default values to defaults.rs

Also we're already have good test for reconnecting producer at crates/fluvio-test/src/tests/reconnection/mod.rs. I just remove the `producer.clear_errors().await;` that is not needed anymore when spu is down.